src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml

```diff
@@ -6,6 +6,10 @@
 					xmlns:ios="http://nventive.com/ios"
 					mc:Ignorable="xamarin android ios">
 
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="MaterialOverride.xaml" />
```

Comment:
I don't think this should be here.

Reply:
When Brushes.xaml is added, all the Material resources should already be added.

Reply:
so this would be a workaround for another issue? I could be looking for another issue, but that means develop will stay broken longer

Reply:
Why do we even redefine brushes like `MaterialPrimaryBrush` here?

Reply:
@kazo0 MaterialBrushes should not be redefined, right? The color override should be what customizes them, right?

Reply:
Also, keep in mind, that `ColorPaletteOverrideSource` has been renamed to `OverrideSource` in later version of Uno.Material

Reply:
yes I saw that too, but we do not want to update just yet. That would be for a future PR (right after this one) Unless you tell me it will fix this issue

Reply:
I am not sure I understand the confusion, are you asking why do we have Brushes.xaml file?

Reply:
Yeah the Brushes.xaml file should be there to add extra non-material brushes used by the app. It should not redefine the material brushes.

Reply:
I tried removing all the brushes that were already define by Material, but now I get an error where PathControl.xaml can not find MaterialOnBackgroundBrush. We normally fix this issue by adding a reference to the ResourceDictionary defining that resource, but now it is not in the app. How should I reference MaterialOnBackgroundBrush now?

Reply:
1. I think one way is to use `MaterialOnBackgroundBrush` directly in your `ControlTemplate` (instead of in a style setter). That kind of sucks.
2. Another way would be to define a new brush based on the same color that can be used in style setters like `MaterialOnBackgroundBrush2`. That also sucks.
3. Ideally we should find a way to use the Material brushes in style setters. Any thoughts @kazo0, @Xiaoy312, @MatFillion?

Reply:
As discussed, solution 2 was implemented to workaround this issue that seem to only touch PathControl styles